### PR TITLE
Dandin87 wthit fix

### PR DESCRIPTION
**Pull request by dandin87**
Advancement Plaques was causing log spam when it was displaying an advancement. I looked at your code guessed it might be a conflict with WTHIT and so linked this code to Deirn and asked if he saw an issue. This was the change he said was needed. Was originally just going to create an issue, but those seem to be disabled on your repo, so figured I might as well PR the change. We are adding the mod to All of Fabric 4 and so figured I could find the log spam issue and get it resolved. Love the mod!

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,7 +47,7 @@ dependencies {
 
 	modApi("me.shedaniel.cloth:cloth-config-fabric:${project.cloth_version}") { exclude(group: "net.fabricmc.fabric-api") }
 
-	modCompileOnly 'curse.maven:wthit-440979:3401714'
+	modCompileOnly "mcp.mobius.waila:wthit-api:fabric-${wthitVersion}"
```
Comment (deirn) [inline build.gradle]: You need the non-API jar to access the internals
```suggestion
	modCompileOnly "mcp.mobius.waila:wthit:fabric-${wthitVersion}"
```

**Review by deirn — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -43,7 +47,7 @@ dependencies {
 
 	modApi("me.shedaniel.cloth:cloth-config-fabric:${project.cloth_version}") { exclude(group: "net.fabricmc.fabric-api") }
 
-	modCompileOnly 'curse.maven:wthit-440979:3401714'
+	modCompileOnly "mcp.mobius.waila:wthit:fabric-${wthitVersion}"
```
Comment (AHilyard) [inline build.gradle]:
```suggestion
	modCompileOnly "mcp.mobius.waila:wthit:fabric-${wthit_version}"
```

File: gradle.properties [hunk 3]
```diff
@@ -13,3 +13,4 @@ author=anthonyhilyard
 version=1.4.0
 fabric_version=0.37.0+1.17
 cloth_version=5.0.34
+wthitVersion=3.9.1
```
Comment (AHilyard) [inline gradle.properties]:
```suggestion
wthit_version=3.9.1
```

**Review by AHilyard — CHANGES_REQUESTED** [review]
Thanks for the PR, I hadn't realized there was a breaking change in WTHIT 3.9.  I only have a small  nitpicky change to maintain naming schemes.  Otherwise this look great.

**Pull request merged**